{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "2331b743_cc4cdaf0",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 3
      },
      "lineNbr": 0,
      "author": {
        "id": 1001712
      },
      "writtenOn": "2025-10-02T08:50:34Z",
      "side": 1,
      "message": "rebasing the patch could make the CI failure go away.",
      "revId": "fef727cca0de5ab7c72c931b316e357c55054918",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4"
    }
  ]
}